routes/web.php

```diff
@@ -1,11 +1,12 @@
 <?php
+
+use App\Http\Controllers\admin\AdminsController;
 use Illuminate\Support\Facades\Route;
 use App\Http\Controllers\HomeController;
 use App\Http\Controllers\ProfileController;
 use App\Http\Controllers\Auth\LoginController;
 use App\Http\Controllers\Admin\UsersController;
 use App\Http\Controllers\ReservationController;
-use App\Http\Controllers\admin\AdminsController;
 use App\Http\Controllers\admin\AttributesController;
```

Comment:
Please "Admin" for the controller capital A, also rename the folder and unify other files